pyproject.toml

```diff
@@ -23,6 +23,7 @@ dependencies = [
     "tldextract",
     "lxml>=4.6.0",
     "defusedxml>=0.7.1",
+    "warcio>=1.7.5",
```

Comment:
Even if it needs to be shipped inside Scrapy it shouldn't add new dependencies to Scrapy as it's optional.

Reply:
1. Figured I would try it first as a PR, to see whether consensus / acceptability developed.  If it's not accepted in the main project, is there a registry of Scrapy plugins somewhere, to improve discoverability?
2. If it is accepted, what's the preferred way to add dependencies for new features?

Reply:
I am unsure how that would be done, but as a third-party extension I guess the question is moot.